pages/blog/release-notes-2.5.0.md

```diff
@@ -0,0 +1,145 @@
+
+We noticed that we overlooked and did not define the Operation Bindings Object in the case of IBM MQ. 
+In AsyncAPI 2.5.0, the definition is empty, but its name is reserved for future use to be consistent with the rest of the bindings.
+
+This change was contributed by [Maciej Urbańczyk](https://github.com/magicmatatjahu). For more detail, see [Maciej's `/spec #840` pull request](https://github.com/asyncapi/spec/pull/840).
+
+## Added new Google Cloud Pub/Sub Bindings
+
+The specification is now extended to support another custom protocol through the bindings feature:
+
+Google Cloud Pub/Sub, thanks to [Jeremy Whitlock](https://github.com/whitlockjc). For more details, check out this [pull request](https://github.com/asyncapi/spec/pull/836) and the [binding definition](https://github.com/asyncapi/bindings/tree/master/googlepubsub).
+
+## Clarifications
+
+Here are notable clarifications that got into the AsyncAPI 2.5.0 release. Here is the list of notable clarifications that may have an impact on the tooling authors:
+
+**ChannelItem.$ref resolution rules**
+
+All references of `MUST be in the format of a URL` sentence have been substituted with `This MUST be in the form of an absolute URL`.
+The meaning of `absolute URL` is now defined by [RFC3986, section 4.3](https://datatracker.ietf.org/doc/html/rfc3986#section-4.3).
+
+**Clarification around URL resolution rules**
+
+The resolution rules around `ChannelItem.$ref` field were not clearly defined. AsyncAPI 2.5.0 comes with explicit clarification that the resolution is done as defined by the [JSON Reference](https://tools.ietf.org/html/draft-pbryan-zyp-json-ref-03).
+
+## Tooling support
+
+The following official AsyncAPI tools are already updated to support the 2.5.0 specification version:
+- JSON Schema that supports validation of AsyncAPI documents is updated in our [specification JSON schema repository](https://github.com/asyncapi/spec-json-schemas). We updated the **@asyncapi/specs** package on NPM to version `3.2`, containing the JSON Schema document for AsyncAPI Spec 2.5.0.
+- [JavaScript Parser](https://github.com/asyncapi/parser-js/) can be used to parse and validate 2.5.0 documents. Upgraded to `the 1.17.0` version.
+- [AsyncAPI React Component](https://github.com/asyncapi/asyncapi-react) allows you to render and validate the documentation of your asynchronous API provided in the AsyncAPI specification format. Upgraded to the `v1.0.0-next.43` version.
+- [HTML template](https://github.com/asyncapi/html-template) uses the latest **@asyncapi/react-component** package. Upgraded to the `0.26.0` version.
+- [JavaScript Converter](https://github.com/asyncapi/converter-js/) enables conversion from any AsyncAPI version into the 2.5.0 version of the spec. Upgraded to the `1.1.0` version.
+- [Generator](https://github.com/asyncapi/generator/) uses the latest @asyncapi/parser package, so while generating output, it can validate 2.5.0 documents. Upgraded to the `1.10.0` version.
+
```

Comment:
I think we should include spectral here if it's updated in time see https://github.com/stoplightio/spectral/pull/2292

cc @magicmatatjahu

Reply:
Spectral is not our tool and also Spectral is releasing one time per day, so I don't think so that we should wait for it and also write about it. It's my opinion, maybe @derberg or @fmvilas will have another.

Reply:
yeah, it is sad it is outside our org and releases

spectral is not our, but asyncapi ruleset is pretty important for community, would be nice to have it included...but yeah, waiting just for it doesn't make sense imho

Reply:
@char0n maybe we can mention that asyncapi ruleset will soon be released and link to the pr?

Reply:
I'm fine with this :) 🚀

Reply:
@derberg see no problem with it. Can you provide suggestion of the change?

Reply:
@char0n have a look https://github.com/asyncapi/website/pull/959/files#r983484253

Reply:
suggestion merged. Thanks!